File: demo/src/main/java/com/training/demo/dto/request/User/UpdateUserRequest.java
package com.training.demo.dto.request.User;

import com.training.demo.utils.enums.RoleType;
import com.training.demo.utils.enums.UserStatus;
import lombok.Getter;
import java.util.Set;

@Getter
public class UpdateUserRequest {
    private String firstName;
    private String lastName;
    private UserStatus status;
    private boolean verifyEmail;
    private Set<RoleType> roles;
}

File: src/app/core/models/request/Users/UpdateUserRequest.ts
import { RoleType } from "src/app/utils/RoleType";
import { UserStatus } from "src/app/utils/UserStatus";

export interface UpdateUserRequest {
  firstName?: string;
  lastName?: string;
  status?: UserStatus;
  verifyEmail?: boolean;
  roles?: RoleType[];
}
